### ISSUE TITLE
UI: Backup Codes setup

### ISSUE THREAD

**Issue by iandunn** [WordPress/wporg-two-factor#39]
We need to add a UI for setting up Backup Codes for the first time. The TOTP screen will redirect to this screen after the user has setup TOTP.

https://github.com/WordPress/two-factor/pull/504 will need to be used locally until it's merged, so that we can use the REST API to interact with the upstream plugin.

See #18 for mockups, may also need to look at WordPress.com for some of the interactions that aren't covered in the screenshots.

This could potentially be done in two PRs, ala [Thinking in React](https://reactjs.org/docs/thinking-in-react.html): one to stub out the UI, and then a second to add dynamic functionality.

**Comment by iandunn** [comment]
There are some minor changes to `backup-codes.js` in #36 that haven't been merged to trunk yet. If anyone picks this up before they are, it'd be good to cherry pick them before starting, to avoid merge conflicts.

**Comment by iandunn** [comment]
To reduce the scope for the MVP, we could skip the copy/print/download buttons, and so we only display the codes. The user could manually do those actions for now.

If we go that route, we should open a new issue for [Iteration 1](https://github.com/WordPress/wporg-two-factor/milestone/2) where we add them.

**Comment by iandunn** [comment]
Refined the UI stubs in 4be26e2
